pkg/watcher/pv_watcher.go

```diff
@@ -29,6 +29,8 @@ import (
 	"github.com/IBM/ibm-csi-common/pkg/utils"
 	"github.com/IBM/ibmcloud-volume-interface/config"
 	"github.com/IBM/ibmcloud-volume-interface/lib/provider"
+	iks_vpc_provider "github.com/IBM/ibmcloud-volume-vpc/iks/provider"
```

Comment:
looks this will create dependency on vpc volume library from vpc file as watcher is a common code

Reply:
As discussed lets move the PV watcher to the individual library code.